.github/actions/setup-terraform/action.yaml

```diff
@@ -38,9 +50,16 @@ runs:
       uses: hashicorp/setup-terraform@v2
 
     - name: Initialize Terraform
-      run: terraform init
+      run: |
+        terraform init \
+          -backend-config "bucket=ccao-terraform-state-us-east-1" \
+          -backend-config "key=terraform.tfstate" \
+          -backend-config "region=us-east-1" \
+          -backend-config "workspace_key_prefix=$REPO/workspaces"
```

Comment:
Terraform technically does not support variables in `backend` configuration, since backends need to be parsed before variables are interpolated, but `-backend-config` is [a common workaround](https://github.com/hashicorp/terraform/issues/13022#issuecomment-294262392) that the community uses to allow for use cases like this one where we need the `workspace_key_prefix` to be dynamically set in order to namespace the state file by repo.